### ISSUE TITLE
_this2.$zendesk.zE is not a function

### ISSUE THREAD

**Issue by Ale-cc** [dansmaculotte/vue-zendesk#4]
![1567047999208](https://user-images.githubusercontent.com/18302674/63907207-27c49d00-ca4d-11e9-815e-ce58668fcc2e.jpg)

In a componet: 
`this.$zendesk.zE('webWidget', 'show')`
here comes a eeor:
But
`this.$zendesk('webWidget', 'show')`
sucess.

**Comment by Ale-cc** [comment]
But sometimes excuse `this.$zendesk('webWidget', 'setLocale', lang[langInCookie || 'en']);` no suceess.
```
const lang = {
        zh: 'zh-cn',
        en: 'en-US',
        ko: 'ko',
      };
this.$zendesk('webWidget', 'setLocale', lang[langInCookie); //  lang[langInCookie] = 'en', but still Chinese
// zE('webWidget', 'setLocale', lang[langInCookie || 'en']); // sucess
```

**Comment by Zevran** [comment]
Hi,

Thanks for reporting this issue !
It should be resolved with 85c0a14ba498025403c2bafbbca23b91cb280e0d

Feel free to reopen it with you still have problems to use `this.$zendesk.zE` directly
